src/main/java/io/pravega/connectors/hadoop/PravegaEventRouter.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.connectors.hadoop;
+
+import java.io.Serializable;
+
+/**
+ * The event router which is used to extract the routing key for pravega from the event.
+ *
+ * @param <T> The type of the event.
+ */
+public interface PravegaEventRouter<T> extends Serializable {
```

Comment:
Please mark this interface as `@FunctionalInterface`

Reply:
(done)